go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -329,10 +357,8 @@ func (c *Controller) syncServices(key string) error {
 							return err
 						}
 						c.serviceTracker.updateService(name, namespace, vip, svcPort.Protocol)
-					}
-					// TODO: Insert a reject ACL if the service doesn't have endpoints associated
```

Comment:
my local tests shows that packets are rejected without having to add ACLs for NodePorts